tests/fake_http_server/ssl_server.py

```diff
@@ -1,8 +1,14 @@
 #!/usr/bin/python3
 from http.server import HTTPServer,SimpleHTTPRequestHandler
+import socket
 import ssl
 
-httpd = HTTPServer(('localhost', 1443), SimpleHTTPRequestHandler)
+class ReUseHTTPServer(HTTPServer):
+    def server_bind(self):
+        self.socket.setsockopt(socket.SOL_SOCKET, socket.SO_REUSEADDR, 1)
+        HTTPServer.server_bind(self)
+
+httpd = ReUseHTTPServer(('localhost', 1443), SimpleHTTPRequestHandler)
```

Comment:
I would love to get rid of these hardcoded ports, but that will probably be a job for another day.

Reply:
Yes, I agree...